config/monitor/config.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package monitor
+
+import (
+	"github.com/crossplane-contrib/terrajet/pkg/config"
+
+	"github.com/crossplane-contrib/provider-jet-azure/apis/rconfig"
+	"github.com/crossplane-contrib/provider-jet-azure/config/common"
+)
+
+const groupLog = "monitor"
+
+// Configure configures monitor group
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("azurerm_monitor_metric_alert", func(r *config.Resource) {
+		r.Kind = "MetricAlert"
+		r.ShortGroup = groupLog
+		r.References = config.References{
+			"resource_group_name": config.Reference{
+				Type: rconfig.APISPackagePath + "/azure/v1alpha1.ResourceGroup",
+			},
+		}
+		r.UseAsync = true
```

Comment:
If, according to your trials, these Cloud resources are provisioned in under our sync context deadlines, we may consider making this synchronous.

Reply:
I tried provisioning/updating/deleting the MetricAlert resource a few times. I did not observe any timeout problem. Provisioning the resource takes about 150 seconds.

So, this line was removed and we made this synchronous.